src/main/webapp/app/exercises/shared/exercise-headers/header-exercise-page-with-details.component.ts

```diff
@@ -106,44 +108,48 @@ export class HeaderExercisePageWithDetailsComponent implements OnChanges, OnInit
      * 1: Publish Results Date
      */
     private setIsNextDueDateExamMode() {
-        this.isNextDueDate = [false, false];
         const now = dayjs();
         if (this.exam?.endDate && now.isBefore(this.exam?.endDate)) {
-            this.isNextDueDate[0] = true;
+            this.nextDueDateIndex = 0;
             this.statusBadges = ['bg-success', 'bg-success'];
         } else if (this.exam?.publishResultsDate && now.isBefore(this.exam?.publishResultsDate)) {
-            this.isNextDueDate[1] = true;
+            this.nextDueDateIndex = 1;
             this.statusBadges = ['bg-danger', 'bg-success'];
         } else {
+            this.nextDueDateIndex = undefined;
             this.statusBadges = ['bg-danger', 'bg-danger'];
         }
     }
 
     /**
      * Determines what element of the header should be highlighted. The highlighted deadline/time is the one being due next
      * Arrays (for badge class (= statusBadges) and highlighting (= isNextDueDate)) consist of
-     * 0: Submission Due Date
-     * 1: Assessment Due Date
-     * 2: Individual Complaint Deadline
-     * 3: Complaint Possible (Yes / No)
+     * 0: Start Date (either release date or start date if set)
+     * 1: Submission Due Date
+     * 2: Assessment Due Date
+     * 3: Individual Complaint Deadline
+     * 4: Complaint Possible (Yes / No)
```

Comment:
Can you add constants for these values and use them in the HTML and the methods?